eox_core/cms/urls.py

```diff
@@ -1,11 +1,17 @@
 """ urls.py """
 
 from django.conf.urls import url, include
-from eox_core.cms.views import management_view, course_team, get_all_orgs
+from eox_core.cms.views import (
+    management_view,
+    course_team,
+    get_all_orgs,
+    get_courses_by_course_regex
```

Comment:
trailing comma.

Are we even running this code through a pycodestyle linter?

Reply:
This repo has no pycodestyle enable in the CircleCI configuration file. @felipemontoya